.github/workflows/inspect_model.yml

```diff
@@ -1,196 +1,61 @@
-name: Inspect model
+name: Inspect Model
 
 on:
-  workflow_dispatch: #add input here
+  workflow_dispatch:
     inputs:
       model_id:
         description: Model ID
         required: false
         type: string
-
   schedule:
-    - cron: '0 */4 * * *'
+    - cron: '0 0 */3 * *' # Runs every 3 days
 
 env:
-  GH_TOKEN: ${{ github.token }}
-  
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
 jobs:
-  publications-and-repositories:
+  inspect-model:
     runs-on: ubuntu-latest
-
     steps:
-      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # pin@v3.5.3
-
-      - name: Add conda to system path
-        run: echo $CONDA/bin >> $GITHUB_PATH
+      - uses: actions/checkout@v3
 
-      - name: Source conda
-        run: source $CONDA/etc/profile.d/conda.sh
-        
-      - name: Set Python to 3.10
-        run:
-         conda install -y python=3.10
-
-      - name: Install dependencies
+      - name: Setup Conda Environment
+        run: |
+          wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
```

Comment:
Same here - can we use conda-incubator action here as well? https://github.com/ersilia-os/ersilia/blob/0685109cbe68ebb3950301e778020c3c6cb87677/.github/workflows/tests_and_cleanup.yml#L64